cluster-commands.go

```diff
@@ -1247,6 +1247,62 @@ func (adm *AdminClient) SiteReplicationResyncOp(ctx context.Context, site PeerIn
 	return res, err
 }
 
+// PoolEndpoints represent endpoints in a given pool
+// along with its setCount and setDriveCount.
+type PoolEndpoints struct {
+	// indicates if endpoints are provided in non-ellipses style
+	Legacy       bool      `json:"legacy"`
+	SetCount     int       `json:"setCount"`
+	DrivesPerSet int       `json:"drivesPerSet"`
+	Endpoints    Endpoints `json:"endpoints"`
```

Comment:
```suggestion
	Drives    Drives `json:"drives"`
```
... that's what it is, correct?

Reply:
yes, will change